core/src/test/scala/org/apache/spark/DistributedSuiteMinTHreads.scala

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
Could you remove this file? This is one of the slowest tests. It's not worth to run it for this issue.